packages/components/src/componentExports.js

```diff
@@ -9,3 +9,4 @@ export { default as VAnalysisFindings } from '@/pages/AnalysisFindings.vue';
 export { default as VFindingDetails } from '@/pages/FindingDetails.vue';
 export { default as VOpenAppmaps } from '@/pages/install-guide/OpenAppMaps.vue';
 export { default as VSidebarSignIn } from '@/components/SignIn.vue';
+export { default as VSerializedSequence } from '@/pages/SerializedSequence.vue';
```

Comment:
Is the `VDiagramSequence` export used anywhere? Can we drop it?

Reply:
I'll look into it